server/lobbyconnection.py

```diff
@@ -585,7 +609,10 @@ def _set_user_agent_and_version(self, user_agent, version):
             irc_pass = "md5:" + str(m.hexdigest())
 
             try:
-                await conn.execute("UPDATE anope.anope_db_NickCore SET pass = %s WHERE display = %s", (irc_pass, login))
+                await conn.execute(
+                    "UPDATE anope.anope_db_NickCore SET pass = %s WHERE display = %s",
+                    (irc_pass, login)
+                )
```

Comment:
:see_no_evil:

Reply:
I like this better than the long line, do you not?

Reply:
Np :-)